tools/stern/src/Stern/Intra.hs

```diff
@@ -324,6 +326,21 @@ deleteBindingTeam tid = do
           . expect2xx
       )
 
+-- | Caution! This may permanently delete all team members!
+deleteBindingTeamForce :: TeamId -> Handler ()
+deleteBindingTeamForce tid = do
+  info $ msg "Deleting team with force flag"
+  g <- view galley
+  void . catchRpcErrors $
+    rpc'
+      "galley"
+      g
+      ( method DELETE
+          . paths ["/i/teams", toByteString' tid]
+          . queryItem' "force" Nothing
```

Comment:
`Nothing`?  Why doesn't this default to `False`?  (We do want `True` for teams with >1 member, right?)

Reply:
`queryItem'` has an optional parameter, and if empty it will just add the query parameter without a value like this:

```
/api/path?force
```

on the API side, existing parameter is interpreted as `true` and non-exisiting as `false`.
it's servants `QueryFlag`

Reply:
Ok.  (I still think having an explicit value is easier to read, though.  I don't see any value in introducing complicated implicit semantics here.)